docs/reference/one-sdk.mdx

```diff
@@ -0,0 +1,134 @@
+---
+slug: '/reference/one-sdk'
+---
+
+# OneSDK for Node.js Reference
+
+OneSDK is a universal API client for Node.js. This page describes OneSDK public API and configuration options.
+
+## Important links
+
+- [OneSDK on GitHub](https://github.com/superfaceai/one-sdk-js) for development and reporting issues
+- [OneSDK on npm](https://www.npmjs.com/package/@superfaceai/one-sdk)
+- [Getting Started](/getting-started) tutorial
+
+## Installation
+
+OneSDK is available as Node package under the name `@superfaceai/one-sdk`. For installation in a Node.js run:
+
+```shell
+npm install @superfaceai/one-sdk
+```
+
+## Usage
+
+### Configuration
+
+OneSDK requires a configuration file, by default located in `superface/super.json` path. You can create this file using the [Superface CLI](https://github.com/superfaceai/cli) by installing a profile and configuring a provider:
+
+```shell
+npx @superfaceai/cli install <profileName>
+npx @superfaceai/cli configure <providerName> -p <profileName>
+```
+
+// TODO: tip where I can find them?
+
+### Initializing the OneSDK client
+
+OneSDK package exports `SuperfaceClient` constructor to initialize an SDK client:
+
+```typescript
+import { SuperfaceClient } from '@superface/one-sdk';
+
+const client = new SuperfaceClient();
+```
+
+### Performing a use-case (`getProfile`, `getUsecase`, `perform`)
+
+With client initialized you can perform the use-cases in installed profiles using the following series of methods:
+
+1. `getProfile('<profileName>')` returns an instance of Profile from the SDK client instance,
+2. `getUsecase('<usecaseName>')` returns a use-case from Profile,
+3. `perform(input, [options])` executes the use-case with provided input data and optional `options` argument.
+
+```typescript
+const profile = await client.getProfile('<profileName>');
+const result = await profile.getUsecase('<usecaseName>').perform({
+  inputField: 1,
+  anotherInputField: 'hello',
+});
+```
+
+Perform returns a [result object](#result-object).
+
+### Selecting a provider (`getProvider`) {#get-provider}
+
+By default, OneSDK picks the first configured provider according to the [`priority` array](/reference/superjson#profilesettings) in `super.json` configuration.
+
+You can get a specific provider with `getProvider` method on SDK client and passing it to the perform:
```

Comment:
"You can set the specific provider in your code using the `getProvider` method ..."